need bundler to run tests

at a minimum, probably need to add a section to the readme on requirements/dependencies necessary for running tests, as well as how to run the tests.

likely because i dont have ruby on my fresh laptop. but again, good to document how to run the tests!

```
~/src/phonegap-docs on master [$] via ⬢ v6.9.1
➔ npm run test

> phonegap-docs@0.0.1 test /Users/maj/src/phonegap-docs
> bundle exec rake test

sh: bundle: command not found

npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/maj/.nvm/versions/node/v6.9.1/bin/node" "/Users/maj/.nvm/versions/node/v6.9.1/bin/npm" "run" "test"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! phonegap-docs@0.0.1 test: `bundle exec rake test`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the phonegap-docs@0.0.1 test script 'bundle exec rake test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the phonegap-docs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bundle exec rake test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs phonegap-docs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls phonegap-docs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/maj/src/phonegap-docs/npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -20,15 +20,37 @@ changed, BrowserSync will regenerate the file and refresh your browser.
 
 ## Running markdownlint
 
-Running markdownlint will return a list of syntax errors that should be fixed before deplying it.
+Running markdownlint will return a list of syntax errors that should be fixed
+before deployment.
 
 ```bash
 npm run lint
 ```
 
+## Tests
+
+### Requirements
+
+A running [ruby](https://www.ruby-lang.org/en/) environment is a must, along
+with a working `gem` command. Ensure you have the `bundle` command available
+by installing bundler, then also retrieve all test-running dependencies via
+`bundle install`:
+
+```bash
+gem install bundler
+bundle install
+```
+
+### Running Tests
+
+```bash
+npm run test
+```
+
 ## Named Anchors
 
-If you want to use in-page navigation you should use named anchors, and not the `id` property of html elements.
+If you want to use in-page navigation you should use named anchors, and not the
+`id` property of html elements.
 
 ```markdown
 * [A link to a different place in the page](#different-place)
@@ -68,8 +90,8 @@ Each commit to the `master` branch is deployed to [docs.phonegap.com](http://doc
 Each commit to the `stage` branch is deployed to [stage.docs.phonegap.com](http://stage.docs.phonegap.com).
 
 Commits to `master` do _not_ need to be staged. Staging is intended to test
-work that may cause issues in production, in particle design focused changes.
-For this reason, the staging branch is mutable. Feel free to delete it or force
+work that may cause issues in production, in particular design focused changes.
+For this reason, the `stage` branch is mutable. Feel free to delete it or force
 push over its history.
 
 ## FAQ

--- a/README.md
+++ b/README.md
@@ -43,6 +43,10 @@ bundle install
 
 ### Running Tests
 
+The tests check the _generated_ documentation, so ensure you are either running
+the local server or have manually generated the documentation via a `npm run
+build` before running the tests!
+
 ```bash
 npm run test
 ```

